communication/sensitive-comms-guidelines.md

```diff
@@ -0,0 +1,26 @@
+
+# Sensitive Communication in the Kubernetes Project
+
+
+1. **Planning and Preparation** \
+
+    1. **Classify Information:**
+        1. **Sensitive:** Information that could harm contributors or the project if disclosed (e.g., personal data, private keys).
+        2. **Important:** Information crucial for project success but not necessarily sensitive (e.g., breaking changes, design documents).
+    2. **Assess Risk:** Evaluate the potential impact of unauthorized access or disclosure, and tailor security measures accordingly.
+    3. **Define Audience:** Specify who needs the information, considering roles and need-to-know principles within the cloud native open-source community & kubernetes project & project consumers.
+    4. **Select Communication Channel:**
+        3. **Sensitive:** private mailing lists, Slack channels or DMs.
+        4. **Important:** Leverage project-approved channels depending on sensitivity, while considering the open-source nature of the project.
+2. **Creating Content**
+    5. **Clarity & Conciseness:** Use plain language, avoiding jargon, to prevent misunderstandings.
```

Comment:
These sub-bullets aren't rendering right, at least with the GitHub rendered view. Though not obvious to me why.